portfolio/src/main/webapp/style.css

```diff
@@ -12,11 +12,11 @@ html {
 
 @keyframes animate {
     from {
-        font-size: 600%;
+        font-size: 100px;
         opacity: 0;
     }
     to {
-        font-size: 810%;
+        font-size: 130px;
```

Comment:
Nice change. Pixels are probably better than percent, for a lot of things. (Although people disagree! If you're curious, read up on font-size and accessibility)